### ISSUE TITLE
Random transcoding for subtitles

### ISSUE THREAD

**Issue by sirev86** [bublikOff/xplay-webos-v2#55]
I know we resolved an issue in this area, but I have observed another issue.. for both types of subtitles (external/internal) the video is transcoded. If i go back to Home..then access the same title, the video is no longer transcoded... the '(T)' next to subtitle name dissapear. 
The app has been restarted.. the tv restarted. Thanks.

**Comment by sirev86** [comment]
https://youtu.be/vjPkb_HnhII
https://youtu.be/BBRr_N1NB_c

Here are 2 videos showing the issues.. it looks like continue watching is still an issue..

**Comment by bublikOff** [comment]
Could you share walking dead with my account? I'll check

**Comment by sirev86** [comment]
Invitation sent.

**Comment by bublikOff** [comment]
Should be OK now.
It was caused because how information about was retrieved about TV.
And in some cases it was "delayed" and the app decided that its not a webOS TV

**Comment by sirev86** [comment]
The subtitle part seems to be resolved.. however the continue watching is not updated.. try to see couple of minutes from ep12..then go back to Home... the lib is not updated.

**Comment by sirev86** [comment]
New issue.. selection of subtitle not working during playback.
It's working from details page.

**Comment by bublikOff** [comment]
Just tried on my PMS
And don see this issue.
Probably to return to the "continue watching" section too fast as PMS puts media to this section after it receives "stop" state for media ... and it may takes PMS some time to put this information to this section

**Comment by sirev86** [comment]
I think you are right.. PMS is  not responding as fast as I would like..
For example,  I am watching a show and on PMS page the continue watching section is not yet updated.

**Comment by sirev86** [comment]
Actually I just tested with the old app and Continue Watching is updated immediately.. so it's not pms fault. 
As i said in othet post.. i'm using the new app with webos 2.0... so maybe the issue is from here?

**Comment by bublikOff** [comment]
Current version has different algorithm if updating home screen. It shows element from cache and only after several seconds it rereqest for new changes. In new it does not cache home screen elements and requests them when you open home screen and does not update them anymore. The same is valid for continue watching an recently added sections in new version

**Comment by bublikOff** [comment]
So the algorithm in current version gives some extra time to PMS to update information in sections on it's side

**Comment by sirev86** [comment]
Ok.. but if PMS is not responding fast enough... then those panels are not updated ... and even if the whole page is manually refreshed, the panels are not refreshed...  perhaps webos 3 is more optimised than 2.0 so that's why is working on your side.
